tests/pool/test_pool.py

```diff
@@ -588,36 +586,20 @@ def test_putconn_wrong_pool(dsn):
                 p2.putconn(conn)
 
 
-def test_del_no_warning(dsn, recwarn):
-    p = pool.ConnectionPool(dsn, min_size=2)
-    with p.connection() as conn:
-        conn.execute("select 1")
-
-    p.wait()
-    ref = weakref.ref(p)
-    del p
-    assert not ref()
-    assert not recwarn, [str(w.message) for w in recwarn.list]
-
-
-@pytest.mark.slow
-def test_del_stop_threads(dsn):
```

Comment:
This too is gone? I don't seem to see a thread that verifies that the workers have stopped correctly (now it should happen after exit.